src/cli/cli.cpp

```diff
@@ -8319,6 +8347,7 @@ template <> otError Interpreter::Process<Cmd("wakeup")>(Arg aArgs[])
             OutputLine("disabled");
         }
     }
+#endif // OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE
```

Comment:
What about extending the `wakeup state` command for WC as well?

Reply:
I'll wait until your PR is merged as there are still discussions about this there.